### PR TITLE
Fix Release v0.1.0 Code review comments

### PR DESCRIPTION
**Pull request by kevinkirkup**

### CODE REVIEW

**Comment by kevinkirkup** [comment]
@dmittendorf This should address all of the PR comments. Can you please review?

File: pureport/resource_azure_connection.go [hunk 1]
```diff
@@ -26,9 +26,9 @@ func resourceAzureConnection() *schema.Resource {
 			Required: true,
 			ForceNew: true,
 		},
-		"peering": {
+		"peering_type": {
 			Type:         schema.TypeString,
-			Description:  "The peering configuration to use for this connection Public/Private",
+			Description:  "The peering type to use for this connection Public/Private",
```
Comment (jmaness) [inline pureport/resource_azure_connection.go]: Should we use a colon to indicate the list of choices and a comma to separate the possible values?

File: pureport/resource_aws_connection_test.go [hunk 2]
```diff
@@ -109,6 +103,8 @@ func TestAWSConnection_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "description", ""),
 					resource.TestCheckResourceAttr(resourceName, "speed", "100"),
 					resource.TestCheckResourceAttr(resourceName, "high_availability", "true"),
+					resource.TestCheckResourceAttr(resourceName, "location_href", "/locations/us-sea"),
+					resource.TestCheckResourceAttr(resourceName, "network_href", "/networks/network-EhlpJLhAcHMOmY75J91H3g"),
```
Comment (jmaness) [inline pureport/resource_aws_connection_test.go]: Just curious, but what is these hardcoded network href for? Not everyone would have access to the same network. Should this value come from some other source?

File: pureport/resource_azure_connection.go (hunk 1 above)
Comment (kevinkirkup) [reply pureport/resource_azure_connection.go]: Can you provide an example?

File: pureport/resource_aws_connection_test.go (hunk 2 above)
Comment (kevinkirkup) [reply pureport/resource_aws_connection_test.go]: This is for the acceptance test environment created in `Dev1`.

File: pureport/resource_azure_connection.go (hunk 1 above)
Comment (jmaness) [reply pureport/resource_azure_connection.go]: "The peering type to use for this connection: Public, Private"

Comment (kevinkirkup) [reply pureport/resource_azure_connection.go]: ok

File: pureport/resource_aws_connection_test.go (hunk 2 above)
Comment (jmaness) [reply pureport/resource_aws_connection_test.go]: Ok, so that's not code that ships? Is there any way to separate test code from the code that ships?

Comment (kevinkirkup) [reply pureport/resource_aws_connection_test.go]: They are two different build targets.
The actual plugin doesn't include the acceptance tests.
This is the same structure for all Terraform Providers.

**Pull request merged**